test/simulation/lntest/node.go

```diff
@@ -88,6 +103,25 @@ func generateListeningPorts() (int, int, int) {
 	return p2p, rpc, rest
 }
 
+// gets unused TCP ports from the system for given count
```

Comment:
a comment in `go` should start with the name being declared ("getFreePorts [...]").